2025-02-26 21:35:30,077 INFO: 调试模式: True [in /data/myblog/app/__init__.py:71]
2025-02-26 21:35:30,078 INFO: 日志文件: /data/myblog/logs/myblog.log [in /data/myblog/app/__init__.py:72]
2025-02-26 21:35:30,095 INFO: 扩展初始化完成 [in /data/myblog/app/__init__.py:85]
2025-02-26 21:35:30,095 INFO: 数据库迁移初始化完成 [in /data/myblog/app/__init__.py:89]
2025-02-26 21:35:30,110 INFO: 蓝图注册完成 [in /data/myblog/app/__init__.py:96]
2025-02-26 21:35:30,111 INFO: 过滤器初始化完成 [in /data/myblog/app/__init__.py:115]
2025-02-26 21:35:30,111 INFO: 应用初始化完成 [in /data/myblog/app/__init__.py:166]
2025-02-26 21:35:30,692 INFO: MyBlog 启动 [in /data/myblog/app/__init__.py:69]
2025-02-26 21:35:30,692 INFO: 配置模式: development [in /data/myblog/app/__init__.py:70]
2025-02-26 21:35:30,692 INFO: 调试模式: True [in /data/myblog/app/__init__.py:71]
2025-02-26 21:35:30,693 INFO: 日志文件: /data/myblog/logs/myblog.log [in /data/myblog/app/__init__.py:72]
2025-02-26 21:35:30,707 INFO: 扩展初始化完成 [in /data/myblog/app/__init__.py:85]
2025-02-26 21:35:30,707 INFO: 数据库迁移初始化完成 [in /data/myblog/app/__init__.py:89]
2025-02-26 21:35:30,723 INFO: 蓝图注册完成 [in /data/myblog/app/__init__.py:96]
2025-02-26 21:35:30,724 INFO: 过滤器初始化完成 [in /data/myblog/app/__init__.py:115]
2025-02-26 21:35:30,724 INFO: 应用初始化完成 [in /data/myblog/app/__init__.py:166]
2025-02-26 21:36:31,410 INFO: 请求: GET http://192.168.72.128:5000/auth/login [in /data/myblog/app/__init__.py:141]
2025-02-26 21:36:31,412 INFO: Locating template 'auth/login.html':
    1: trying loader of application 'app'
       class: jinja2.loaders.FileSystemLoader
       encoding: 'utf-8'
       followlinks: False
       searchpath:
         - /data/myblog/app/templates
       -> found ('/data/myblog/app/templates/auth/login.html') [in /home/ops/.local/lib/python3.12/site-packages/flask/debughelpers.py:178]
2025-02-26 21:36:31,491 INFO: 请求: GET http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:141]
2025-02-26 21:36:31,492 WARNING: 页面未找到: http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:109]
2025-02-26 21:36:31,492 INFO: Locating template 'errors/404.html':
    1: trying loader of application 'app'
       class: jinja2.loaders.FileSystemLoader
       encoding: 'utf-8'
       followlinks: False
       searchpath:
         - /data/myblog/app/templates
       -> found ('/data/myblog/app/templates/errors/404.html') [in /home/ops/.local/lib/python3.12/site-packages/flask/debughelpers.py:178]
2025-02-26 21:36:31,493 INFO: Locating template 'blog/base.html':
    1: trying loader of application 'app'
       class: jinja2.loaders.FileSystemLoader
       encoding: 'utf-8'
       followlinks: False
       searchpath:
         - /data/myblog/app/templates
       -> found ('/data/myblog/app/templates/blog/base.html') [in /home/ops/.local/lib/python3.12/site-packages/flask/debughelpers.py:178]
2025-02-26 21:36:34,056 INFO: 请求: POST http://192.168.72.128:5000/auth/login [in /data/myblog/app/__init__.py:141]
2025-02-26 21:36:34,182 INFO: 请求: GET http://192.168.72.128:5000/admin/ [in /data/myblog/app/__init__.py:141]
2025-02-26 21:36:34,185 INFO: Locating template 'admin/index.html':
    1: trying loader of application 'app'
       class: jinja2.loaders.FileSystemLoader
       encoding: 'utf-8'
       followlinks: False
       searchpath:
         - /data/myblog/app/templates
       -> found ('/data/myblog/app/templates/admin/index.html') [in /home/ops/.local/lib/python3.12/site-packages/flask/debughelpers.py:178]
2025-02-26 21:36:34,187 INFO: Locating template 'admin/layout.html':
    1: trying loader of application 'app'
       class: jinja2.loaders.FileSystemLoader
       encoding: 'utf-8'
       followlinks: False
       searchpath:
         - /data/myblog/app/templates
       -> found ('/data/myblog/app/templates/admin/layout.html') [in /home/ops/.local/lib/python3.12/site-packages/flask/debughelpers.py:178]
2025-02-26 21:36:34,230 INFO: 请求: GET http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:141]
2025-02-26 21:36:34,230 WARNING: 页面未找到: http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:109]
2025-02-26 21:36:36,528 INFO: 请求: GET http://192.168.72.128:5000/admin/users/ [in /data/myblog/app/__init__.py:141]
2025-02-26 21:36:36,535 INFO: Locating template 'admin/user/list.html':
    1: trying loader of application 'app'
       class: jinja2.loaders.FileSystemLoader
       encoding: 'utf-8'
       followlinks: False
       searchpath:
         - /data/myblog/app/templates
       -> found ('/data/myblog/app/templates/admin/user/list.html') [in /home/ops/.local/lib/python3.12/site-packages/flask/debughelpers.py:178]
2025-02-26 21:36:36,576 INFO: 请求: GET http://192.168.72.128:5000/static/js/user.js [in /data/myblog/app/__init__.py:141]
2025-02-26 21:36:36,589 INFO: 请求: GET http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:141]
2025-02-26 21:36:36,590 WARNING: 页面未找到: http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:109]
2025-02-26 21:36:41,365 INFO: 请求: GET http://192.168.72.128:5000/admin/users/2/edit [in /data/myblog/app/__init__.py:141]
2025-02-26 21:36:41,370 INFO: Locating template 'admin/user/edit.html':
    1: trying loader of application 'app'
       class: jinja2.loaders.FileSystemLoader
       encoding: 'utf-8'
       followlinks: False
       searchpath:
         - /data/myblog/app/templates
       -> found ('/data/myblog/app/templates/admin/user/edit.html') [in /home/ops/.local/lib/python3.12/site-packages/flask/debughelpers.py:178]
2025-02-26 21:36:41,427 INFO: 请求: GET http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:141]
2025-02-26 21:36:41,428 WARNING: 页面未找到: http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:109]
2025-02-26 21:36:46,371 INFO: 请求: GET http://192.168.72.128:5000/admin/users/ [in /data/myblog/app/__init__.py:141]
2025-02-26 21:36:46,402 INFO: 请求: GET http://192.168.72.128:5000/static/js/user.js [in /data/myblog/app/__init__.py:141]
2025-02-26 21:36:46,415 INFO: 请求: GET http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:141]
2025-02-26 21:36:46,415 WARNING: 页面未找到: http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:109]
2025-02-26 21:36:55,336 INFO: 请求: GET http://192.168.72.128:5000/admin/users/2/edit [in /data/myblog/app/__init__.py:141]
2025-02-26 21:36:55,393 INFO: 请求: GET http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:141]
2025-02-26 21:36:55,394 WARNING: 页面未找到: http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:109]
2025-02-26 21:36:58,902 INFO: 请求: POST http://192.168.72.128:5000/admin/users/2/edit [in /data/myblog/app/__init__.py:141]
2025-02-26 21:36:58,913 INFO: 更新结果: {'status': 'success', 'message': '用户更新成功', 'user': <User test>} [in /data/myblog/app/controllers/admin/user.py:137]
2025-02-26 21:37:00,444 INFO: 请求: GET http://192.168.72.128:5000/admin/users/ [in /data/myblog/app/__init__.py:141]
2025-02-26 21:37:00,479 INFO: 请求: GET http://192.168.72.128:5000/static/js/user.js [in /data/myblog/app/__init__.py:141]
2025-02-26 21:37:00,490 INFO: 请求: GET http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:141]
2025-02-26 21:37:00,491 WARNING: 页面未找到: http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:109]
2025-02-26 21:37:07,892 INFO: 请求: GET http://192.168.72.128:5000/admin/users/ [in /data/myblog/app/__init__.py:141]
2025-02-26 21:37:07,933 INFO: 请求: GET http://192.168.72.128:5000/static/js/user.js [in /data/myblog/app/__init__.py:141]
2025-02-26 21:37:07,943 INFO: 请求: GET http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:141]
2025-02-26 21:37:07,944 WARNING: 页面未找到: http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:109]
2025-02-26 21:37:09,409 INFO: 请求: GET http://192.168.72.128:5000/admin/users/2/edit [in /data/myblog/app/__init__.py:141]
2025-02-26 21:37:09,463 INFO: 请求: GET http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:141]
2025-02-26 21:37:09,464 WARNING: 页面未找到: http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:109]
2025-02-26 21:37:14,483 INFO: 请求: POST http://192.168.72.128:5000/admin/users/2/edit [in /data/myblog/app/__init__.py:141]
2025-02-26 21:37:14,492 INFO: 更新结果: {'status': 'success', 'message': '用户更新成功', 'user': <User test>} [in /data/myblog/app/controllers/admin/user.py:137]
2025-02-26 21:37:15,741 INFO: 请求: GET http://192.168.72.128:5000/admin/users/ [in /data/myblog/app/__init__.py:141]
2025-02-26 21:37:15,780 INFO: 请求: GET http://192.168.72.128:5000/static/js/user.js [in /data/myblog/app/__init__.py:141]
2025-02-26 21:37:15,792 INFO: 请求: GET http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:141]
2025-02-26 21:37:15,792 WARNING: 页面未找到: http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:109]
2025-02-26 21:37:27,934 INFO: 请求: GET http://192.168.72.128:5000/admin/users/1/edit [in /data/myblog/app/__init__.py:141]
2025-02-26 21:37:27,985 INFO: 请求: GET http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:141]
2025-02-26 21:37:27,986 WARNING: 页面未找到: http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:109]
2025-02-26 21:37:37,428 INFO: 请求: GET http://192.168.72.128:5000/admin/users/ [in /data/myblog/app/__init__.py:141]
2025-02-26 21:37:37,475 INFO: 请求: GET http://192.168.72.128:5000/static/js/user.js [in /data/myblog/app/__init__.py:141]
2025-02-26 21:37:37,489 INFO: 请求: GET http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:141]
2025-02-26 21:37:37,490 WARNING: 页面未找到: http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:109]
